packages/next/build/webpack/plugins/middleware-plugin.ts

```diff
@@ -311,14 +343,21 @@ function getCodeAnalyzer(params: {
     } = params
     const { hooks } = parser
 
+    function allowDynamicCodeEvaluation() {
```

Comment:
There is no (valid) way to get the entrypoint config during parsing. This check must be delayed until the full graph is available.